cal/cal/models.py

```diff
@@ -584,6 +584,21 @@ def query(self, calendar_ids=None, start=None, end=None):
         return events_qs
 
 
+    def queryWeek(self, calendar_ids=None, start=None, end=None):
+        queryset = []
```

Comment:
The `query()` function is so named because it returns a Django `QuerySet` object. To prevent our code from getting really confusing, your function should either return a `QuerySet` object or not be named `query_xxx`. `get_events` looks like the closest analogy since you are returning a Python list, and `get_events` returns a set.

The reason we want to namespace like this is so that we have a reasonable expectation for when we want to do more complicated operations in the future like python set joins/intersections and/or database level joins and intersections.

My recommendation is to change the return type to a set and change this to `get_events_week` or something